src/org/exist/indexing/StreamListener.java

```diff
@@ -43,6 +43,13 @@
         STORE,
 
         /**
+         * Mode for replacing the nodes of a document with another document
+         * this is really a group mode, which is later followed by {@link #REMOVE_ALL_NODES}
+         * and then {@link #STORE}
+         */
+        REPLACE_DOCUMENT,
```

Comment:
Ah I see, sorry I thought you were saying the opposite. Okay I need to make a small change so that is:
```java
final StreamListener listener = broker.getIndexController().getStreamListener();
listener.endReplaceDocument(transaction);
```